.github/workflows/build_rocks.yaml

```diff
@@ -214,7 +248,7 @@ jobs:
       - name: Upload rock to ghcr.io
         if: steps.rock-cache.outputs.cache-hit != 'true' || inputs.cache-action == 'save'
         run: |
-          skopeo --insecure-policy copy oci-archive:$(ls "${{ matrix.path }}"/*.rock) docker://$IMAGE_REF --dest-creds "${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}"
+          skopeo --insecure-policy copy oci-archive:$(ls "${{ matrix.rock.path }}"/*.rock) docker://$IMAGE_REF --dest-creds "${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Hello, it seems that skopeo isn't available for ARM64, so this step also needs to be fixed.

Reply:
I think it should be available on ARM64 https://packages.ubuntu.com/jammy/arm64/skopeo/download